### PR TITLE
kde-apps/kio-extras: activities packages PlasmaActivities, Qt6Sql, PlasmaActivitiesStats are required & added missing dependency dev-libs/qcoro

### PR DESCRIPTION
**Pull request by rachid-a**
Signed-off-by: Rachid Achellal <rachid@achellal.com>

kde-apps/kio-extras: activities packages PlasmaActivities, Qt6Sql, PlasmaActivitiesStats are required & added missing dependency dev-libs/qcoro

### CODE REVIEW

**Comment by Chiitoo** [comment]
Same as here:

https://github.com/gentoo/kde/pull/948#issuecomment-1856750972

File: kde-apps/kio-extras/kio-extras-9999.ebuild [hunk 1]
```diff
@@ -41,11 +42,9 @@ DEPEND="
 	>=kde-frameworks/kxmlgui-${KFMIN}:6
 	>=kde-frameworks/solid-${KFMIN}:6
 	>=kde-frameworks/syntax-highlighting-${KFMIN}:6
-	activities? (
-		>=dev-qt/qtbase-${QTMIN}:6[sql]
```
Comment (a17r) [inline kde-apps/kio-extras/kio-extras-9999.ebuild:45]: Where did that dependency go?

**Comment by a17r** [comment]
Please read up a bit on git and how to do rebasing/commit squashing - it's easy! ;)

Again, #gentoo-dev-help lends a helping hand.

Meanwhile I'm trying to get something upstream: https://invent.kde.org/network/kio-extras/-/merge_requests/320

**Comment by rachid-a** [comment]
thanks i will look into the rebasing/commit squashing

**Pull request closed without merging**